reference/Blaseball-API.v1.yaml

```diff
@@ -2680,13 +2693,28 @@ components:
         * 151: Decree narration
         * 152: Will results
         * 154: Team shamed
+        * 153: Team stat adjustment (Lineup Batting adjusted, Shadow Defense adjusted)
         * 155: Team shames
         * 156: Sun 2 grants win
         * 157: Black Hole swallows
         * 158: Eliminated from postseason
         * 159: Postseason advance
+        * 161: Player gained blood type
+        * 165: High Pressure (The pressure is on/off, Overperforming added and removed) 
         * 166: Lineup optimized
+        * 168: Peanut allergy cured
```

Comment:
this is only for The Nut Button, peanut misters cure peanut allergies using type 72